core/ir/instr_shared.c

```diff
@@ -470,6 +470,25 @@ instr_get_category(instr_t *instr)
 /* in rest of file, directly de-reference for performance (PR 622253) */
 #define instr_get_category inlined_instr_get_category
 
+const char *
+instr_get_category_name(dr_instr_category_t category)
+{
+    switch (category) {
+    case DR_INSTR_CATEGORY_UNCATEGORIZED: return "uncategorized";
```

Comment:
Add a (non-Doxygen) comment by the dr_instr_category_t enum as a reminder to update this function when adding new categories.

Reply:
Done.